.circleci/make-rpm-packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "PARAM_RELEASE: $PARAM_RELEASE"
+echo "PARAM_DIST: $PARAM_DIST"
+
+if test "x$PARAM_RELEASE" = "x"; then
+    echo "Env variable PARAM_RELEASE is not set! For example PARAM_RELEASE=8, for CentOS 8"
+    exit 1
+elif test "x$PARAM_DIST" = "x"; then
+    echo "Env variable PARAM_DIST is not set! For example PARAM_DIST=centos"
+    exit 1
+fi
+
+if [ "$PARAM_DIST" = centos ]; then
+  if [ "$PARAM_RELEASE" = 8 ]; then
+      dnf install -y 'dnf-command(config-manager)'
+      yum config-manager --set-enabled PowerTools
+      yum install -y diffutils python3-sphinx
+  else
+      yum install -y python-sphinx
+  fi
+fi
+
+yum install -y epel-release
+yum install -y automake jemalloc-devel git libedit-devel libtool libunwind-devel make pcre-devel python3 sudo rpm-build yum-utils
+
+
+export DIST_DIR=build
+
+# XXX: we should NOT have to do that here, they should be in the
+# spec as BuildRequires
+yum install -y make gcc
+
+adduser varnish
+chown -R varnish:varnish /workspace
+
+cd /varnish-cache
+rm -rf $DIST_DIR
+mkdir $DIST_DIR
+
+
+echo "Untar redhat..."
+tar xavf /workspace/redhat.tar.gz -C $DIST_DIR
+
+echo "Untar orig..."
+tar xavf /workspace/varnish-*.tar.gz -C $DIST_DIR --strip 1
+
+echo "Build Packages..."
+# use python3
+sed -i '1 i\%global __python %{__python3}' "$DIST_DIR"/redhat/varnish.spec
```

Comment:
I still hate that one, now that `centos:8` is out we may have a better case. Or even better, since `python3` is a requirement now, we may not even need it anymore.

I'll check

Reply:
redhat/varnish.spec has this: `Requires:  python(abi) >= 3.4`
It seems python3 is a hard requirement.

Reply:
I know :-) but i still had issues last time I tried to change the packaging, I just can't remember if it was pre or post `python3` migration

Reply:
See my PR in pkg-varnish-cache. I've moved this line directly in varnish.spec and the build is green.